tests/basic/customer_issues_test.go

```diff
@@ -1077,7 +1077,6 @@ var _ = Describe("{FADAPodRecoveryAfterBounce}", func() {
 	var k8sCore = core.Instance()
 
 	itLog := "FADAPodRecoveryAfterBounce"
-	stepLog := "App Stuck in ContainerCreation State with Device Exists in the backend"
 	It(itLog, func() {
 		log.Infof(stepLog)
```

Comment:
I think you need to delete this also if stepLog deletion is intentional

Reply:
done